src/core_plugins/kibana/public/management/index.js

```diff
@@ -32,7 +32,8 @@ uiModules
     transclude: true,
     scope: {
       sectionName: '@section',
-      omitPages: '@omitBreadcrumbPages'
+      omitPages: '@omitBreadcrumbPages',
+      breadcrumbPageTitle: '='
```

Comment:
I'd probably just call this `page-title`, since the consumer shouldn't care where the title is used, and we could move that title out of the breadcrumb in the future.